util/config.go

```diff
@@ -203,7 +204,11 @@ func readCert(reader io.Reader) (contents string) {
 }
 
 func getPackageIdentifierFromConfig(config *Config) string {
```

Comment:
this func returns package name for `OneginiConfigModel.java` and `SecurityController.java`